package/MDAnalysis/topology/TPRParser.py

```diff
@@ -66,6 +66,7 @@
    122        28             2021                 yes
    127        28             2022                 yes
    129        28             2023                 yes
+   133        28             2024.1               yes
```

Comment:
Ok, I added the generation as `28` based on a fresh compilation of GROMACS `2024.1` with this adjustment:

```diff
--- a/src/gromacs/fileio/tpxio.cpp
+++ b/src/gromacs/fileio/tpxio.cpp
@@ -1034,6 +1034,7 @@ static void do_inputrec(gmx::ISerializer* serializer, t_inputrec* ir, int file_v
 
     ir->tpxFileVersion = file_version;
 
+    fprintf(stderr, "****** Note: tpx_generation %d\n", tpx_generation);
     if (file_version != tpx_version)
     {
         /* Give a warning about features that are not accessible */
```

Then used one of the `.tpr` files here as follows:

`gmx mdrun -s dummy_2024.tpr`

which includes in its output:

```
<snip>
Reading file dummy_2024.tpr, VERSION 2024.1-spack (single precision)
****** Note: tpx_generation 28
****** Note: tpx_generation 28
<snip>
```